agent/src/cli/root.ts

```diff
@@ -18,3 +19,4 @@ export const rootCommand = new Command()
     .addCommand(codyBenchCommand)
     .addCommand(chatCommand())
     .addCommand(authCommand())
+    .addCommand(tscContextRetrieverCommand())
```

Comment:
Let's create an `internal` subcommand that groups internal/testing features like this one. We may move the `bench` command to there as well.
```suggestion
    .addCommand(new Command('internal').addCommand(tscContextRetrieverCommand()))
```
I'm working on making the cli a useful tool for end-users so I want to keep the toplevel commands clean. Users should be able to run `cody help` and see a meaningful list of commands.

Reply:
done